File: yoyo_last/src/Entite_kl/Offres.java

package Entite;

import java.sql.Date;
import java.util.ArrayList;

/**
 *
 * @author asus
 */

public class Offres {
    private int id_offre ;

    public int getId_offre() {
        return id_offre;
    }

    public void setId_offre(int id_offre) {
        this.id_offre = id_offre;
    }

    private int id_prop; 
    private String nom ;
    private Date  date_debut;
    private Date date_fin ;
    private String description ; 
    private String prix ; 
    private int id_centre ;
        private int id_reservation ; 
    private ArrayList<Reservation> Reservations;

    public Offres(String nom) {
        this.nom = nom;
    }
    
   
   public Offres(){
        Reservations = new ArrayList<>();
   
       
   }

    public Offres(String nom, Date date_debut, Date date_fin, String description, String prix) {
        this.nom = nom;
        this.date_debut = date_debut;
        this.date_fin = date_fin;
        this.description = description;
        this.prix = prix;
    }

    
   

    
    public int getId_reservation() {
        return id_reservation;
    }

    public void setId_reservation(int id_reservation) {
        this.id_reservation = id_reservation;
    }

    public int getId_prop() {
        return id_prop;
    }

    public void setId_prop(int id_prop) {
        this.id_prop = id_prop;
    }


    public String getNom() {
        return nom;
    }

    public Date getDate_debut() {
        return date_debut;
    }

    public Date getDate_fin() {
        return date_fin;
    }

    public String getDescription() {
        return description;
    }

   

    public int getId_centre() {
        return id_centre;
    }

    public void setNom(String nom) {
        this.nom = nom;
    }

    public void setDate_debut(Date date_debut) {
        this.date_debut = date_debut;
    }

    public void setDate_fin(Date date_fin) {
        this.date_fin = date_fin;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public String getPrix() {
        return prix;
    }

    public void setPrix(String prix) {
        this.prix = prix;
    }

   

    public void setId_centre(int id_centre) {
        this.id_centre = id_centre;
    }

    @Override
    public String toString() {
        return "Offres{ ID:" + id_offre + " nom=" + nom + ", date_debut=" + date_debut + ", date_fin=" + date_fin + ", description=" + description + ", prix=" + prix + ", id_centre=" + id_centre + '}';
    }
    
    
    
}

File: pidev1/src/AppBundle/Entity/Offres.php
<?php

namespace AppBundle\Entity;

/**
 * Offres
 */
class Offres
{
    /**
     * @var \DateTime
     */
    private $dateDebut;

    /**
     * @var \DateTime
     */
    private $dateFin;

    /**
     * @var string
     */
    private $descrOffre;

    /**
     * @var float
     */
    private $prix;

    /**
     * @var string
     */
    private $nom;


    /**
     * Set dateDebut
     *
     * @param \DateTime $dateDebut
     *
     * @return Offres
     */
    public function setDateDebut($dateDebut)
    {
        $this->dateDebut = $dateDebut;

        return $this;
    }

    /**
     * Get dateDebut
     *
     * @return \DateTime
     */
    public function getDateDebut()
    {
        return $this->dateDebut;
    }

    /**
     * Set dateFin
     *
     * @param \DateTime $dateFin
     *
     * @return Offres
     */
    public function setDateFin($dateFin)
    {
        $this->dateFin = $dateFin;

        return $this;
    }

    /**
     * Get dateFin
     *
     * @return \DateTime
     */
    public function getDateFin()
    {
        return $this->dateFin;
    }

    /**
     * Set descrOffre
     *
     * @param string $descrOffre
     *
     * @return Offres
     */
    public function setDescrOffre($descrOffre)
    {
        $this->descrOffre = $descrOffre;

        return $this;
    }

    /**
     * Get descrOffre
     *
     * @return string
     */
    public function getDescrOffre()
    {
        return $this->descrOffre;
    }

    /**
     * Set prix
     *
     * @param float $prix
     *
     * @return Offres
     */
    public function setPrix($prix)
    {
        $this->prix = $prix;

        return $this;
    }

    /**
     * Get prix
     *
     * @return float
     */
    public function getPrix()
    {
        return $this->prix;
    }

    /**
     * Get nom
     *
     * @return string
     */
    public function getNom()
    {
        return $this->nom;
    }
}

